internal/connect/command_custom_plugin_version_create.go

```diff
@@ -33,15 +33,15 @@ func (c *customPluginCommand) newVersionCreateCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Create custom connector plugin version for plugin "ccp-123456".`,
-				Code: "confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --beta=true --sensitive-properties=passwords,keys,tokens",
```

Comment:
What's the reason for removing this from the examples? Do we no longer support it?

Reply:
Hi Brian, this is to resolve error while running `go run cmd/lint/main.go -aff-file cmd/lint/en_US.aff -dic-file cmd/lint/en_US.dic`
```
For context "Cloud", 6 errors occurred:
	* confluent connect custom-plugin version create: flag `--beta` must not use "=" in example 1
	* confluent connect custom-plugin version create: flag `--sensitive-properties` must not use "=" in example 1
	* flag usage doesn't end with punctuation for --beta on `confluent connect custom-plugin version create`
	* confluent connect custom-plugin version update: flag `--beta` must not use "=" in example 1
	* confluent connect custom-plugin version update: flag `--sensitive-properties` must not use "=" in example 1
	* flag usage doesn't end with punctuation for --beta on `confluent connect custom-plugin version update`

exit status 1
```

Reply:
cmd/lint/main.go allows you to override some of these lints if they don't make sense. For instance, we necessarily need to write `--beta=true` here. Don't sacrifice the quality of the examples to make the linter pass!!